owncloudApp/src/main/java/com/owncloud/android/authentication/PatternManager.java

```diff
@@ -99,28 +109,23 @@ private void checkPattern(Activity activity) {
         activity.startActivity(i);
     }
 
-    private boolean patternShouldBeRequested() {
-        int PATTERN_TIMEOUT = 1000;
-        if ((SystemClock.elapsedRealtime() - timeStamp) > PATTERN_TIMEOUT &&
-                mVisibleActivitiesCounter <= 0
-        ) {
-            return isPatternEnabled();
-        }
-        return false;
-    }
-
-    public boolean isPatternEnabled() {
-        SharedPreferences appPrefs = PreferenceManager.getDefaultSharedPreferences(MainApp.Companion.getAppContext());
-        return appPrefs.getBoolean(PatternLockActivity.PREFERENCE_SET_PATTERN, false);
+    public void onBiometricCancelled(Activity activity) {
+        // Ask user for pattern
+        checkPattern(activity);
```

Comment:
Not very important and I know that it was there, but sometimes this kind of commentary can be removed by using a descriptive function name f.e. `askUserForPattern(activity)`